bach/bach/series/series.py

```diff
@@ -125,8 +125,6 @@ def __init__(
         name: str,
         expression: Expression,
         group_by: Optional['GroupBy'],
-        sorted_ascending: Optional[bool],
-        index_sorting: List[bool],
         instance_dtype: StructuredDtype,
         order_by: Optional[List[SortColumn]] = None,
```

Comment:
request: could we make this non-optional? I have the feeling that having this optional just hides places where we forget to call this correctly.

```suggestion
        order_by: List[SortColumn],
```

Additionally, this will save like a line of code in this function

Reply:
mmm has the same typing as `DataFrame.__init__`. Was aiming more for consistency.

Reply:
Matching what the docstring says  https://github.com/objectiv/objectiv-analytics/blob/17c87cea3ea87cca0dcb6842a7d6b77d30025ee3/bach/bach/dataframe.py#L192

Reply:
okay I think we disagree. I just think it is slightly better, because it saves a bit of code and is less ambiguous between functions. You feel different, that's fine. Let's go with the changes you have made.